package.json

```diff
@@ -93,20 +58,18 @@
     "buffer": "^6.0.3",
     "eslint": "^8.55.0",
     "eslint-plugin-jest": "^27.2.2",
-    "highlight.js": "^11.9.0",
     "husky": "^8.0.3",
     "jest": "^29.5.0",
     "jest-environment-jsdom": "^29.6.1",
-    "lit": "^3.1.0",
-    "material-icons": "^1.13.12",
-    "minimatch": "^9.0.3",
-    "open-props": "^1.6.16",
-    "parcel": "^2.10.3",
+    "parcel": "^2.11.0",
     "rimraf": "^5.0.5",
+    "rollup": "^4.9.6",
+    "rollup-plugin-filesize": "^10.0.0",
+    "sass": "^1.70.0",
     "semantic-release": "^22.0.10",
     "stylelint": "^16.0.2",
     "stylelint-config-rational-order": "^0.1.2",
-    "stylelint-order": "^6.0.4"
```

Comment:
Secretly removed?

Reply:
Good catch, I was testing some stuff for the demo page and forgot to put it back.
> Or maybe I secretly hate that it is always complaining about the order of my CSS attributes...